config/item.asset

```diff
@@ -1,8 +1,7 @@
-@BaseClass = item: "An Item"
+@include "data.asset"
+
+@BaseClass base(data) = item : "An item"
 [
-  Id(string) : "Id" : "Item Id"
-  Name(string) : "Name" : "Item Name"
   Description(string) : "Description" : "This is my item"
-  Model(string): "Model" : ""
   HoldType(int): "HoldType" : "1"
```

Comment:
We need to use the choices feature to have a dropdown list
syntax: 
	something2(choices) : "second number" : 0 : "Your choice of numbers!" =
	[
		0 : "Default"
		1 : "Something"
		2 : "Another Thing"
	]

source : [https://developer.valvesoftware.com/wiki/FGD](https://developer.valvesoftware.com/wiki/FGD)

Reply:
To be ticket